bgpd/bgpd.h

```diff
@@ -602,6 +602,9 @@ struct bgp {
 	uint32_t default_connect_retry;
 	uint32_t default_delayopen;
 
+	/* BGP minimum holdtime.  */
+	uint32_t default_min_holdtime;
```

Comment:
uint16

Reply:
holdtime and keepalive are defined in _uint32_t_.

Reply:
But 65k is limited to uin16_t, what's the reason to use uint32_t?

Reply:
Fixed in b042667